### PR TITLE
Uni v3 mai usdc

### PR DESCRIPTION
**Pull request by abtheo**
Sync complete @ https://thegraph.com/hosted-service/subgraph/abtheo/otter-development
image

### CODE REVIEW

File: src/utils/TreasuryRevenue.ts [hunk 1]
```diff
@@ -98,6 +100,25 @@ export function updateTreasuryRevenueClaimUsdplusReward(block: BigInt, claim: Cl
   treasuryRevenue.save()
 }
 
+export function updateTreasuryRevenueClaimUsdcReward(block: BigInt, claim: ClaimReward): void {
```
Comment (0xthk) [inline src/utils/TreasuryRevenue.ts:105]: We should separate MAI and USDC fee in UniV3 strategy. Using `ClaimToken` event should be easier.

Comment (abtheo) [reply src/utils/TreasuryRevenue.ts:105]: Ah okay, latest commit adds Mai and Usdc to Revenue. 

What about the impermanent loss?  I'd rather use the "realised" profit for the APR calculation in the Investments page, so should I use PayoutReward there?

Comment (0xthk) [reply src/utils/TreasuryRevenue.ts:105]: Yes, `PayoutReward` is staker actually get and `ClaimToken` is revenue from the strategy.

**Comment by 0xthk** [comment]
tried `maiMarketValue` and `usdcMarketValue`, both are zero 🤔

**Comment by abtheo** [comment]
> tried `maiMarketValue` and `usdcMarketValue`, both are zero 🤔

Fixed this issue, but now the Kyberswap revenue is broken 😅

Resyncing now. Need to double check the total daily revenue is correct before merging

**Comment by 0xthk** [comment]
> > tried `maiMarketValue` and `usdcMarketValue`, both are zero 🤔
> 
> Fixed this issue, but now the Kyberswap revenue is broken 😅
> 
> Resyncing now. Need to double check the total daily revenue is correct before merging

huh, why KNC/LDO affects by the fixes? 😅

**Comment by abtheo** [comment]
@0xthk  Sync complete, ready for review & merge;
image
image

**Pull request merged**